Dockerfile

```diff
@@ -0,0 +1,38 @@
+# base image
+FROM python:slim-buster as base
+# update Image
+RUN apt-get update
+# install poetry
+RUN pip install poetry
+# copy all except in dockerignore
+COPY . .
+# install prerequisits
+RUN poetry install --no-root --no-dev
+RUN pip install -r requirements.txt
+# use port 5000 from container
+EXPOSE 5000
+
+
+
+# prod image
+FROM base as production
+# install gunicorn
+RUN pip install gunicorn
+# Cmd /entrypoint
+CMD ["gunicorn"  , "-b", "0.0.0.0:5000", "todo_app.app:create_app()"]
+
+# dev image
+FROM base as development
+# install flask
+RUN pip install flask
```

Comment:
Similarly you shouldn't need to explicitly install flask, poetry should already have done so

Reply:
This was ok to build with Run pip install flask hashed out